### ISSUE TITLE
Dashboard page elements need more contrast

### ISSUE THREAD

**Issue by cndouglas** [brave/brave-browser#3051]
## Description 
The text and buttons on the dashboard sometimes are hard to see. In particular, the white buttons at the bottom right blend in with light colored backgrounds.

I recommend that a dark shadow be added around the text/buttons to provide sufficient contrast for both light and dark backgrounds.

## Steps to Reproduce
For Brave 0.60.16 dev.
   1. Open a new tab.
   2. Refresh the page until you get a light background, like the "beach" background.
   3. Try to read all the text and buttons.

## Actual result:
The "Estimated Time Saved" text and Bookmarks button are hard to see:
![](https://user-images.githubusercontent.com/19424103/51564137-fe5ae780-1e53-11e9-8283-18546d807204.png)

## Expected result:
All the text and buttons should be visible.

## Reproduces how often: 
Sometimes, with certain backgrounds.

## Brave version (brave://version info)
Brave | 0.60.16 Chromium: 72.0.3626.64 (Official Build) dev(64-bit)
Revision | eaa9668e80ce5405e7f1902579558ea725c06ca1-refs/branch-heads/3626@{#708}
OS | Mac OS X

### Reproducible on current release:
- Does it reproduce on brave-browser dev/beta builds? Yes, 0.60.16 dev.

### Website problems only:  
- Does the issue resolve itself when disabling Brave Shields? Not applicable.
- Is the issue reproducible on the latest version of Chrome? Not applicable.

**Comment by rebron** [comment]
Closing. Fixed with https://github.com/brave/brave-browser/issues/6904 to account for contrast and readability.
